torchtune/models/clip/_model_builders.py

```diff
@@ -1,6 +1,6 @@
 from torchtune.models.clip._transforms import CLIPImageTransform
 
-def _clip_vit_224_transform():
+def clip_vit_224_transform():
```

Comment:
There is zero reason for a builder to exist and not be public. It's not a utility but a model. The only reason it's not add to the docs yet is because there should be a model builder as well, not just the transform.